ci-operator/step-registry/baremetalds/e2e/ovn/router/baremetalds-e2e-ovn-router-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: baremetalds-e2e-ovn-router
+  steps:
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v6
+        NETWORK_TYPE=OVNKubernetes 
+    pre:
+      - ref: baremetalds-packet-setup
+      - ref: ipi-install-rbac
+      - ref: ovn-conf
```

Comment:
Same point here (as for dualstack config)

Reply:
+1. Changed.